src/common/mfu_daos.c

```diff
@@ -5427,7 +5423,7 @@ int cont_deserialize_all_props(struct hdf5_args *hdf5,
     }
 
     entry = &prop->dpp_entries[17];
-    rc = cont_deserialize_prop_roots(hdf5, entry, "DAOS_PROP_CO_ROOTS");
+    rc = cont_deserialize_prop_roots(hdf5, entry, "DAOS_PROP_CO_ROOTS", roots);
```

Comment:
Does this function use both entry and roots?

Reply:
@daltonbohning yes, the entry->dpe_val_ptr points to the roots struct. For structs in the DAOS properties you have point these to some other memory (the special property struct like acl, roots, etc) you can't store it directly in properties like you can with dpe_val.